src/commands/psql.rs

```diff
@@ -12,6 +15,31 @@ pub async fn psql<'x>(cli: &mut Client<'x>, _options: &Options)
     match cli.params.get::<PostgresAddress>() {
         Some(addr) => {
             let mut cmd = Command::new("psql");
+            #[cfg(all(feature="dev_mode", unix))]
+            {
+                if let Some(dir) = option_env!("PSQL_DEFAULT_PATH") {
+                    let psql_path = Path::new(dir).join("psql");
+                    if !psql_path.exists() {
+                        eprintln!("WARNING: {} does not exists",
+                                  psql_path.display());
+                    }
+                    let npath = if let Some(path) = env::var_os("PATH") {
+                        if path.is_empty() {
+                            dir.into()
+                        } else {
+                            let mut npath = OsString::with_capacity(
+                                dir.len() + 1 + path.len() + 1);
+                            npath.push(dir);
+                            npath.push(":");
```

Comment:
Is there an equivalent of [os.pathsep](https://docs.python.org/3/library/os.html#os.pathsep) in Rust?

Reply:
Fixed to `env::join_paths`. I wanted to skip this code on windows (because we don't have windows build on edgedb as far as I know), but nevermind.